### ISSUE TITLE
a11y: Review and update image descriptions.

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#1152]
For images across the documentation, review the following.

- Has appropriate alt text.

  Text should be brief and descriptive of the visual appearance of the image. Do not include abstract descriptions, purpose, or fine detail, unless critically important to understanding the image. Try to limit to 7 words or 30 characters.

- Has a description in body text immediately preceding or following the image.

  The descriptive text should relate the content of the image to the purpose for the image existing at this point in the documentation. It isn't enough to merely describe the image visual content.

- Transcribes text content in the image, if the text content is relevant to the purpose of the image.

  Screenshots of terminals, code, emails, etc., should have their text transcribed into the body text. For text longer than a few words, or written attributable to another source, use a block quote. For code and terminal contents, use a code block.

- Has appropriate contrast within the image.

  Key elements of the image should have sufficient contrast with their background. This applies not only to text, but other image features. If key content lacks sufficient contrast and the content is within our control, it should be corrected and the image reacquired. If the content is outside our control, then the content must be described in text. Key content includes, but is not limited to, GUI controls and labels, text intended to be read, and image annotations.
